backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Services/FileService.java

```diff
@@ -40,6 +50,15 @@ public FileService(FileRepository fileRepository) {
     public List<CustomFile> getFilesAtRoot() {
         List<CustomFile> files = fileRepository.findAllByParentIsNull();
         logger.debug("Retrieved {} files and folders located at root level.", files.size());
+
+        // Add log to InfluxDB
+        String operation = "RetrieveFilesAtRoot";
```

Comment:
Unsure if this is supposed to be the method's name or just a small description.
If the first one you should guarantee that it uses the same name as its method (if you want to use strings).
Otherwise you can try to get the method's name (not sure what the implications are with this)